packages/editor-ui/src/components/CodeNodeEditor/baseExtensions.ts

```diff
@@ -35,6 +33,7 @@ export const baseExtensions = [
 		{ key: 'Enter', run: insertNewlineAndIndent },
 		{ key: 'Tab', run: acceptCompletion },
 		{ key: 'Enter', run: acceptCompletion },
+		{ key: 'Cmd-/', run: toggleComment },
```

Comment:
does this work for windows too?

Reply:
Switched to `Mod` and [confirmed](https://n8nio.slack.com/archives/C036PA5D42D/p1665661545077129?thread_ts=1665659962.461969&cid=C036PA5D42D).